docs/snippets/gettingStarted/webApi/apiBDD.e2e.spec.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import {
+  getInMemoryEventStore,
+  type EventStore,
+} from '@event-driven-io/emmett';
+import {
+  ApiE2ESpecification,
+  expectResponse,
+  getApplication,
+} from '@event-driven-io/emmett-expressjs';
+import { randomUUID } from 'node:crypto';
+import { beforeEach, describe, it } from 'node:test';
+import type { PricedProductItem } from '../events';
+import { ShoppingCartStatus } from './shoppingCart';
+import { shoppingCartApi } from './simpleApi';
+
+const getUnitPrice = () => {
+  return Promise.resolve(100);
+};
+
+describe('ShoppingCart E2E', () => {
+  let clientId: string;
+  let shoppingCartId: string;
+
+  beforeEach(() => {
+    clientId = randomUUID();
```

Comment:
Notice that we don't need the `uuid()` library to generate uuid v4 - it's already supported by node's native "crypto" module

Reply:
Ah, cool. For those needs, this will be sufficient. Happy to remove the dependency on `uuid` package!